extensions/script-libraries/oauth2/oauth2.livecodescript

```diff
@@ -1,4 +1,4 @@
-﻿script "com.livecode.library.oauth2"
+script "com.livecode.library.oauth2"
```

Comment:
What's the reason for the change to the first line? (My guess is that a BOM has been added or removed)

Reply:
Yeap, it seems it you edit a `.livecodescript` file in the IDE it removes the BOM.

I have fixed it now